README.md

```diff
@@ -724,6 +724,7 @@ A curated list of awesome Java frameworks, libraries and software.
 
 * [Apache Tapestry](http://tapestry.apache.org/) - Component-oriented framework for creating dynamic, robust, highly scalable web applications.
 * [Apache Wicket](http://wicket.apache.org/) - Component-based web application framework similar to Tapestry with a stateful GUI.
+* [Baratine](http://baratine.io) - Asynchronous Java framework with unified single-thread & data model for standalone or embedded(.jar) high performing REST & WebSocket Microservices.
```

Comment:
We try to be as concise as possible, therefore I suggest the following description, of course you can modify it:

`Toolkit for building distributed and reactive applications for multiple environments, either standalone or embedded.`